resources/assets/js/features/user/actionTypes.js

```diff
@@ -0,0 +1,4 @@
+export const REGISTER_SUCCESS = 'REGISTER_SUCCESS';
```

Comment:
add USER_ prefix to all actions, since you register them under user component, an actions describes user entity state changing